boards/arm/mec172xevb_assy6906/support/spi_cfg.txt

```diff
@@ -0,0 +1,50 @@
+; Everglades SPI Image Generator configuration file
+[SPI]
+SPISizeMegabits = 128
+Flashmap = false
+FlshmapAddr = 0
+
+[DEVICE]
+TagAddr0 = 0
+TagAddr1 = 0
+BoardID = 0x316
```

Comment:
I noticed the layout for SPI configuration file is slightly different than MEC15xx one.
Could you add a small comment here here to know if this field is mandatory or informative only?

Reply:
Added a comment. BoardID is part of Boot-ROM PlatformID OTP feature.  If PlatformID is enabled and a value put in OTP the Boot-ROM will check BoardID in SPI image and only load if they match. PlatformID is disabled by default in OTP.